plugins/fileservices/fileservices.cpp

```diff
@@ -3128,3 +3128,49 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+bool getDefaultValue(const char * processName, const char * propertyName, StringBuffer & strDefaultValue)
+{
+    Owned<IConstEnvironment> daliEnv = openDaliEnvironment();
```

Comment:
This shouldn't be called in (is undefined) in containerized version.

Reply:
Yes, that is true. I move it into a conditional block.